src/libAtomVM/globalcontext.h

```diff
@@ -215,6 +240,42 @@ void globalcontext_send_message(GlobalContext *glb, int32_t process_id, term t);
  */
 void globalcontext_send_message_nolock(GlobalContext *glb, int32_t process_id, term t);
 
+#ifndef AVM_NO_TASK_DRIVER
```

Comment:
Let's add for clarity, for 3rd party components, AVM_HAS_SAFE_TASK_FUNCTIONS or something like that. That can be checked so we don't fail with some missing function at build time.

Reply:
Don't you think we should change the name of AVM_NO_TASK_DRIVER instead to something positive, e.g. AVM_TASK_DRIVERS_ENABLED?

Reply:
yes, sounds a good idea.